coeus-impl/src/main/java/org/kuali/coeus/common/budget/impl/personnel/BudgetPersonValuesFinder.java

```diff
@@ -94,6 +94,15 @@ private String getPersonUniqueKey(BudgetPerson budgetPerson) {
     	uniqueKey.append(budgetPerson.getEffectiveDate());
     	return uniqueKey.toString();
     }
+    
+    protected String getBudgetPersonLabel(BudgetPerson budgetPerson) {
+    	StringBuffer personLabel = new StringBuffer();
+    	personLabel.append(budgetPerson.getPersonName());
```

Comment:
Yeah, again verified this scenario, did not see a case where it goes null. 
Will add additional check.